docs/UserGuide.md

```diff
@@ -54,35 +55,92 @@ not be so obvious!
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`c-list`** : Lists all employees.
+   * **`list`** : Lists all contacts.
```

Comment:
I think here should change back to what it was. And I spot some of the similar problems in the PR. I will mark them all here, so you can easily refer to that line and change those back. Thanks 😄